Openshift-EE/pipelines/ElasticSearch/stages/3-functional/86TL-App-replica-scaleup/app-replica-scaleup

```diff
@@ -59,13 +59,19 @@ sed -i '/command:/i \
 echo "Running the litmus test for Elasticsearch Deployment Scaleup.."
 cat run_scaleup_test.yml
 
-bash ../Openshift-EE/utils/litmus_job_runner label='app: elasticsearch-deployment-litmus' job=run_scaleup_test.yml
+bash ../Openshift-EE/utils/litmus_job_runner label='app:elasticsearch-deployment-litmus' job=run_scaleup_test.yml
 cd ..
 bash Openshift-EE/utils/dump_cluster_state;
 bash Openshift-EE/utils/event_updater jobname:app-replica-scaleup $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
 
 if [ "$?" != "0" ]; then
-## python3 Openshift-EE/utils/result/result_update.py $job_id 86TL 3-functional "Scale the statefulset application replicas and verify the ring configuration" Fail $pipeline_id "$current_time" $commit_id $gittoken
+##python3 Openshift-EE/utils/result/result_update.py $job_id 86TL 3-functional "Scale the statefulset application replicas and verify the ring configuration" Fail $pipeline_id "$current_time" $commit_id $gittoken
```

Comment:
remove the comment and let the result be updated

Reply:
Comment is removed